src/main/java/calculator/operator/NumberOperator.java

```diff
@@ -0,0 +1,44 @@
+package calculator.operator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class NumberOperator {
+
+    private Integer value;
+
+    public NumberOperator(Integer value) {
+        validateNegative(value);
+        this.value = value;
+    }
+
+    public Integer getValue() {
+        return value;
+    }
+
+    public Integer sum(NumberOperator number) {
+        value += number.getValue();
+        return value;
+    }
```

Comment:
리턴 값도 NumberOperator 객체 면 어떨까요? NumberOperator 객체를 필드로 받아 Integer를 리턴하는 것이 조금 어색하다고 느껴져요.

Reply:
```suggestion
    public Integer sum(NumberOperator number) {
        value += number.value;
        return value;
    }
```
getter를 사용하지 않는 방법도 있겠네요. 이번 과정에서는 getter 사용을 최대한 자제해 보는 것은 어떨까요?

Reply:
제거 하였습니다~!

Reply:
getter를 안쓰는게 조금 어렵네요ㅠㅠ 노력해보겠습니다ㅠㅠ